### PR TITLE
multivariate_gaussian extension code

### PR DESCRIPTION
**Pull request by williamjameshandley**
# Description

This PR implements marginalisation and conditioning functions for multivariate_gaussian models, for both mixture and non-mixture models.

# Checklist:

- [X] I have performed a self-review of my own code
- [X] My code is PEP8 compliant (`flake8 lsbi tests`)
- [X] My code contains compliant docstrings (`pydocstyle --convention=numpy lsbi`)
- [X] New and existing unit tests pass locally with my changes (`python -m pytest`)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have appropriately incremented the [semantic version number](https://semver.org/) in both README.rst and lsbi/_version.py

### CODE REVIEW

File: lsbi/model.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def logP(self, theta):
     def logL(self, theta):
         """P(D|theta) as a scalar."""
         return (self.logLmax
-                + multivariate_normal.logpdf(theta, self.mu_L, self.Sigma_L)
+                + multivariate_normal(self.mu_L, self.Sigma_L).logpdf(theta)
```
Comment (yallup) [inline lsbi/model.py:282]: Does this cache properly? I saw there was some changes to cached properties but depending on the intended usage of this, is the frozen dict rebuilt every time?

**Review by yallup — APPROVED** [review]
Looks good, the conditioning utility for mixture_multivariate_normal is a very neat addition!

File: lsbi/model.py (hunk 1 above)
Comment (williamjameshandley) [reply lsbi/model.py:282]: I think I've now removed all cached properties, so this should behave as expected.

**Comment by williamjameshandley** [comment]
Just pushed a bugfix correcting the conditioning for mixture models, and added a test that shows this explicitly.

**Pull request merged**